gensim/models/word2vec.py

```diff
@@ -1116,17 +1116,36 @@ def doesnt_match(self, words):
         dists = dot(vectors, mean)
         return sorted(zip(dists, words))[0][1]
 
-    def __getitem__(self, word):
+    @classmethod
```

Comment:
This `sum` doesn't belong to core gensim -- it's a single `numpy.mean` command, it can't get simpler than that.